helper-cli/src/main/kotlin/commands/ListCopyrightsCommand.kt

```diff
@@ -45,11 +48,19 @@ internal class ListCopyrightsCommand : CommandWithHelp() {
     )
     private lateinit var ortResultFile: File
 
+    @Parameter(
+        description = "A file containing garbage copyright statements entries which are to be ignored.",
+        names = ["--copyright-garbage-file"],
+        order = PARAMETER_ORDER_OPTIONAL
+    )
+    private var copyrightGarbageFile: File? = null
+
     override fun runCommand(jc: JCommander): Int {
         val ortResult = ortResultFile.expandTilde().readValue<OrtResult>()
+        val copyrightGarbage = copyrightGarbageFile?.expandTilde()?.readValue<CopyrightGarbage>().orEmpty()
 
         val copyrightStatements = ortResult
-            .getProcessedCopyrightStatements()
+            .getProcessedCopyrightStatements(omitExcluded = true, copyrightGarbage = copyrightGarbage.items)
```

Comment:
`true` is the default value for `omitExcluded`, you could omit it.

Reply:
Done.